pkg/openstack/loadbalancer.go

```diff
@@ -1180,6 +1197,9 @@ func (lbaas *LbaasV2) ensureOctaviaPool(lbID string, name string, listener *list
 	if pool == nil {
 		createOpt := lbaas.buildPoolCreateOpt(listener.Protocol, service, svcConf, name)
 		createOpt.ListenerID = listener.ID
+		if svcConf.supportLBTags {
+			createOpt.Tags = append(createOpt.Tags, lbaas.getCustomLoadBalancerTags(service)...)
```

Comment:
I think this is empty by default, no need to use `append`, you can just set it. Now I wonder - could we just put this inside `buildPoolCreateOpt` instead?